script.js

```diff
@@ -0,0 +1,16 @@
+document.querySelectorAll('li').forEach((item) => {
+  const span = document.createElement('span');
```

Comment:
It is not a good way to modify DOM.
You need it in js only for some dynamic changes.
Add static layout directly to html